### DISCUSSION TITLE
Azure App Service - Urers & Data Deleting After Restarting App in Azure

### DISCUSSION THREAD

**Discussion by josh99smith** [Budibase/budibase#8110] [Help]
![Capture](https://user-images.githubusercontent.com/4806827/193665498-00fede8d-6ec2-4ddd-bb56-5209fffa2767.JPG)

Hello,

I recently migrated my app over to self-hosted on Azure using the new Azure App Service feature. Everything went super smooth, installed with no issues. However, when I went to restart my app/vm in Azure, I loaded up Budibase and it was acting like a fresh install. Asking for me to setup a new Admin account like its a fresh install of Budibase. I tested this multiple times but still got the same result. Restarting the app, or stopping/starting the server causes it to revert to default install state.  Any ideas what could be causing the instance of Budibase to revert back to default state every time I reset the app/server?

**Comment by melohagan** [comment]
Hey @josh99smith 

The data for Budibase will be stored in docker volumes. So Azure might be wiping your volumes?

You can run a backup via the CLI now: https://docs.budibase.com/docs/backups

**Comment by ethanreynolds57** [comment]
I'm also new to docker containers in AAS and am encountering the same behavior. Did you ever happen to find a fix or correction?
